github/github.go

```diff
@@ -880,6 +880,23 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+func formatDuration(d time.Duration) string {
+	secondsTotal := int(d.Seconds())
+	hours := int(secondsTotal / 60 / 60)
+	minutes := int((secondsTotal - hours*60*60) / 60)
+	seconds := int(secondsTotal - hours*60*60 - minutes*60)
+
+	if hours > 0 {
+		return fmt.Sprintf("%dh%02dm%02ds", hours, minutes, seconds)
+	}
```

Comment:
As far as I know, GitHub rate limit resets at most 1 hour away from current time. So, I think we can get rid of this case for `hours > 0` because it'll never be true.

Even if it's true in some rare case, it's fine for us to print that rate limit is going to be reset in 120 minutes.